services/core-api/app/api/verifiable_credentials/resources/w3c_map_credential_resource.py

```diff
@@ -0,0 +1,101 @@
+from json import dumps
+from datetime import datetime
+from flask import current_app, request
+from werkzeug.exceptions import Forbidden
+from flask_restx import Resource, reqparse
+from app.api.utils.include.user_info import User
+
+from app.config import Config
+from app.extensions import api
+
+from app.api.utils.resources_mixins import UserMixin
+from app.api.services.traction_service import TractionService
+from app.api.verifiable_credentials.manager import VerifiableCredentialManager
+from app.api.mines.permits.permit_amendment.models.permit_amendment import PermitAmendment
+
+from app.api.utils.feature_flag import Feature, is_feature_enabled
+
+PRESENT_PROOF = "present_proof"
+CONNECTIONS = "connections"
+CREDENTIAL_OFFER = "issue_credential"
+OUT_OF_BAND = "out_of_band"
+PING = "ping"
+ISSUER_CREDENTIAL_REVOKED = "issuer_cred_rev"
+
+
+class W3CCredentialResource(Resource, UserMixin):
+
+    @api.doc(description='Endpoint to get vc by uri, including guid.', params={})
+    def get(vc_guid: str):
+        return "Hello World"
+
+
+class W3CCredentialListResource(Resource, UserMixin):
+    parser = reqparse.RequestParser(trim=True)
+    parser.add_argument(
+        'permit_amendment_guid',
+        type=str,
+        help='GUID of the permit amendment.',
+        location='json',
+        store_missing=False)
+
+    @api.expect(parser)
+    @api.doc(
+        description=
+        "returns a signed w3c credential for a specific permit_amendment, using new aca-py endpoints, but cannot use public did."
+    )
+    def post(self):
```

Comment:
Could we add in a role check for these endpoints? 
I assume they should only be available for Minespace proponents and certain core users. Just so we don't accidentally forget and leave them open when we eventually turn on the feature flag

Reply:
Good catch. not sure what it will be. but i'll make it MINE_EDIT for now. 

Minespace likely won't be interacting with this directly. most likely it will be celery tbh.

Reply:
Makes sense, sounds good to me